Update to add to explanations around what merge cluster is for, and a…

ehumber opened the pull request:
…dd 2 topics, one consumer example at the end

framiere commented on merge-cluster/Readme.md:
```diff
@@ -17,166 +28,194 @@ Conduktor Proxy's merge cluster feature help single kafka clients can work with
-The below creates two topics:
-* master_topic on master cluster
-* secondary_topic on secondary cluster
+Create a topic on each of the backing kafka clusters, connecting to each individual Kafka cluster directly and creating topics using the Kafka console tools.  The `eurpoe_cars` topic is created on the cluster with hostname`kafka1_m`, which is running on port 9092, and `us_cars` is created on cluster with hostname `kafka1_s1`.
```
typo, `europe_cars`

framiere commented on merge-cluster/Readme.md:
```diff
@@ -2,7 +2,18 @@
+
+Test applications can read live production data from a production cluster, for realistic testing, without affecting or altering that production data, and then use test topics on a test cluster to write data, all without having to reconfigure any applications.  There is no need to do expensive replication of data to get useful and accurate testing data.  
+
+Failover is made much simpler by Conduktor Gateway managing the routing in your Kafka environment.  Say you want to do a managed failover from cluster A to cluster B as part of planned maintenance.  If your applications connect to Conduktor Gateway, to do this move just alter the Gateway configuration, and your applications are routed to the new cluster, with no need for application restarts or c ...
```
your application connects (with a s?)

framiere reviewed (approved):
❤️ ❤️

manhcuongbk56 reviewed (approved):
LGTM

Stu-conduktor commented on merge-cluster/Readme.md:
````diff
@@ -17,166 +28,194 @@ Conduktor Proxy's merge cluster feature help single kafka clients can work with
 docker-compose up -d 
 ```
 
+Note:  You might see the error message `! conduktor-proxy The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested`.
````
warning rather than error? @ehumber 
I assumed this was when running on apple ARM processors, so is fine as it's emulated.

Stu-conduktor commented on merge-cluster/Readme.md:
```diff
@@ -17,166 +28,194 @@ Conduktor Proxy's merge cluster feature help single kafka clients can work with
-The below creates two topics:
-* master_topic on master cluster
-* secondary_topic on secondary cluster
+Create a topic on each of the backing kafka clusters, connecting to each individual Kafka cluster directly and creating topics using the Kafka console tools.  The `europe_cars` topic is created on the cluster with hostname`kafka1_m`, which is running on port 9092, and `us_cars` is created on cluster with hostname `kafka1_s1`.
```
If being picky you might want a space after hostname and before `kafka1_m'

ehumber commented:
@manhcuongbk56 I did a rebase off main to get latest commits, and that messed up the PR into your branch so I've changed the PR target to be main here.


Could you double check you're happy with the last two commits - the readme shouldn't be controversial as it's just tidy up, but for the docker compose I added memory limits to the containers as during testing I hit a loops that ate about 4GB of memory in the proxy, and realised that we didn't have any limits (except what docker is allowed to use).  

With 9 JVM based docker containers running, that's quite a heavy ask of the system it is running on so it seemed sensible to hard limit everything but proxy to half a GB.  

That also gives users an indication of how much memory they need if they choose to run this.

framiere commented:
```
$docker pull ghcr.io/conduktor/conduktor-proxy:em11
Error response from daemon: manifest unknown
```
Am I missing something? And if not, can you publish your image somewhere?

ehumber commented:
> ```
> $docker pull ghcr.io/conduktor/conduktor-proxy:em11
> Error response from daemon: manifest unknown
> ```
> 
> Am I missing something? And if not, can you publish your image somewhere?

https://github.com/conduktor/conduktor-proxy/pull/443/files needs merging then I'll update the image in this PR once I've published those changes to dockerhub (that's mentioned in the docker-compose as a comment, but probably hard to spot, sorry!)

framiere commented:
Built your branch, and it works perfectly.
Nicely done!

manhcuongbk56 commented:
sorry for missing this, @ehumber . It is perfect, :D .
